content/writer.go

```diff
@@ -133,3 +133,12 @@ func (cw *writer) Close() (err error) {
 
 	return nil
 }
+
+func (w *writer) Truncate(size int64) error {
+	if size != 0 {
+		return errors.New("Truncate: unsupported size")
```

Comment:
In the future I would assume supporting this would entail rehashing all the data up to this point.